src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -19,7 +19,7 @@ origin_trial:
 {% Aside %}
   We're currently working on this API as part of the new
   [capabilities project](https://developers.google.com/web/updates/capabilities).
-  Starting in Chrome 73, it is available as an [origin trial](#ot) on Android.
+  Starting in Chrome 73, the API is available as an [origin trial](#ot) on Android.
```

Comment:
I'm the one who put 'it' in the sentence because we already said 'this API' in the previous one.

Reply:
That actually creates an ambiguous pronoun error though. The most recent prior singular noun is _project_, so _it_ would technically be referring to that, which can make comprehension harder. (I actually had to read the sentence a second time.)